test/notification/apsProperties.js

```diff
@@ -711,6 +711,32 @@ describe("Notification", function() {
       });
     });
 
+    describe("interruption-level", function() {
+      it("defaults to undefined", function() {
+        expect(compiledOutput()).to.not.have.nested.property("aps.interruption\-level");
+      });
+
+      it("can be set to a string", function() {
+        note.interruptionLevel = "the-interruption-level";
+
+        expect(compiledOutput()).to.have.nested.property("aps.interruption\-level", "the-interruption-level");
+      });
+
+      it("can be set to undefined", function() {
+        note.interruptionLevel = "the-interruption-level";
+        note.interruptionLevel = undefined;
+
+        expect(compiledOutput()).to.not.have.nested.property("aps.interruption\-level");
+      });
+
+      describe("setInterruptionLevel", function () {
+        it("is chainable", function () {
+          expect(note.setInterruptionLevel("the-interruption-level")).to.equal(note);
+          expect(compiledOutput()).to.have.nested.property("aps.interruption\-level", "the-interruption-level");
```

Comment:
```suggestion
          expect(compiledOutput()).to.have.nested.property("aps.interruption-level", "the-interruption-level");
```

nit: unnecessary escaping